creativecloud/scripts/utils.js

```diff
@@ -59,21 +59,22 @@ function getDecorateAreaFn() {
   };
 
   async function loadLCPImage(area = document, { fragmentLink = null } = {}) {
-    const firstBlock = area.querySelector('body > main > div > div');
+    const marquee = area.querySelector('.marquee');
+    const interactiveMarquee = area.querySelector('.interactive-marquee');
     let fgDivs = null;
     switch (true) {
-      case firstBlock?.classList.contains('changebg'): {
-        firstBlock.querySelector(':scope > div:nth-child(1)').querySelectorAll('img').forEach(eagerLoad);
+      case marquee != null:
+        eagerLoad(marquee.querySelector('img'));
```

Comment:
This assumes that the marquee is using an actual image. I know there have been usages of CSS-gradient based marquees that were later changed back to image-based ones. Maybe we should add a check here for the presence of an `img` to account for that?

Reply:
Never mind, the `eagerLoad` function already checks for null so we're good here.